tests/upgrade/tests.bats

```diff
@@ -269,58 +270,38 @@ stageOlvm() {
 	ocne cluster show -C $(yq -e .name $CLUSTER_CONFIG) -f "config.providers.olvm"
 }
 
-@test "Basic Kubernetes Tests for 1.26" {
```

Comment:
Why are these being removed?

Reply:
I restored the upgrade starting point of Kubernetes 1.26.